tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -2689,11 +2689,16 @@ TEST_P(MatmulFusionTest, Llama2FFN) {
   const FusionKernelRuntime* runtime =
       executor_cache.getMostRecentKernelRuntime();
 
-  EXPECT_TRUE(runtime->isSegmented());
-
   if (fusion_enabled) {
-    EXPECT_EQ(runtime->fusionSegments()->groups().size(), 2);
+    if (isOptionEnabled(EnableOption::FuseMultipleMatmuls)) {
```

Comment:
Running `NVFUSER_ENABLE=fuse_multiple_matmuls test_matmul --gtest_filter=MatmulFusionTest.Llama2FFN/fuse` will result in a kernel similar to that of `test_matmul --gtest_filter=MatmulSchedulerTest.Llama2FFNNoSegmentation`